Release Notes - Active Development

BrandonJenkins14 opened the pull request:
Active Development release to go out concurrent w/ v1.8.0. 

Includes:
- Release Notes for @latest CLI and CICS plug-in 
- 2 new plug-ins - FTP and SCS plug-in and corresponding install docs
- Web help 

TODO: update TPSRs 

Note that I haven't receive any info about MQ, IMS, or Db2 changes. Assuming there are none but let me know if anything to add.

BrandonJenkins14 commented:
@awharn @tjohnsonBCM Anything else to mention here that is _unique_ in @latest version of core/imperative, CICS, Db2, IMS, or MQ?

awharn commented:
I am unsure about the other plugins, but there have been quite a few changes to CICS that only appear in the latest version, including:

- Defining, Enabling, Installing, Discarding, Disabling, and Deleting URIMaps
    - Define - https://github.com/zowe/zowe-cli-cics-plugin/pull/55
    - Delete - https://github.com/zowe/zowe-cli-cics-plugin/pull/62
    - Enable & Disable - https://github.com/zowe/zowe-cli-cics-plugin/pull/64
    - Install & Discard - https://github.com/zowe/zowe-cli-cics-plugin/pull/71
- Defining and Deleting webservices
    - Define - https://github.com/zowe/zowe-cli-cics-plugin/pull/65
    - Options to Associate with URIMaps - https://github.com/zowe/zowe-cli-cics-plugin/pull/66
    - Delete - https://github.com/zowe/zowe-cli-cics-plugin/pull/67
- Adding and removing CSD Groups to/from CSD Lists - https://github.com/zowe/zowe-cli-cics-plugin/pull/72
- Defaulting CICS Plugin to connect to CMCI via HTTPS - https://github.com/zowe/zowe-cli-cics-plugin/pull/86
- Defaulting defining URIMAP to use HTTPS - https://github.com/zowe/zowe-cli-cics-plugin/pull/87

zFernand0 commented on docs/getting-started/summaryofchanges.md:
```diff
@@ -15,13 +15,20 @@ Welcome to Zowe Active Development documentation.
 
 **Features and enhancements:**
 
+- The `zowe files copy` command was added for copying the contents of a data set or member to another data set or member. [#580](https://github.com/zowe/zowe-cli/pull/580)
```
Was this the only human change in this PR?
If so, I think we need to do something with the `.vuepress`  folder.

BrandonJenkins14 replied:
Good morning @zFernand0  - I added the new web help files (.zip, HTML, and PDF). This causes many files to be changed. Can you clarify what the issue is? I plan to build and test the web help links shortly, then also add the LTS web help in another PR.

zFernand0 replied:
I was just wondering if most of these files are in fact generated by a script, it might be worth sourcing them on a separate repo (or somewhere else) so that it doesn't affect the PR review process. i.e. do something so that people can see what you want them to see 😄 

@jackjia-ibm, any suggestions?

BrandonJenkins14 replied:
Would make sense if it's possible - and it's probably unnecessary to have the doc-site broken link tests scan these files. 

@zFernand0 I can also show you the steps I take to add these files, as I'm sure that updating them could be automated. But that's another story.

tjohnsonBCM commented on docs/.vuepress/public/web_help/docs/zowe.html:
```diff
@@ -15,21 +15,28 @@ <h2><a href="zowe.html" name="zowe">zowe</a></h2>
+<li><a href="zowe_cics.html">cics</a> - Interact with IBM CICS programs and              transactions.</li>
+<li><a href="zowe_config.html">config</a> - Manage configuration and overrides              </li>
+<li><a href="zowe_db2.html">db2</a> - Interact with IBM Db2 for z/OS                  </li>
+<li><a href="zowe_endevor.html">endevor</a> - CA Endevor SCM plug-in for listing Endevor       environment information, working with elements and packages located in specified Endevor instance.</li>
```
The web help on zowe.org should not contain the Broadcom plugins (Endevor, FMP, OPS, Extended Files, Extended Jobs).

BrandonJenkins14 commented:
@zFernand0 @awharn  @MarkAckert I added simple Bill of Materials entries for FTP and SCS plug-ins (Active Dev). 

@awharn I added some notes about the CICS features. Let me know if sufficient - perhaps I should add basic command syntax for them here like the previous section.

MarkAckert commented on docs/appendix/bill-of-materials.md:
```diff
@@ -67,14 +67,16 @@ The following materials are distributed with Zowe&trade; binaries:
-| zowe-db2-*.tgz | Zowe CLI Plug-in for IBM® Db2® for z/OS® | [zowe-cli-db2-plugin](https://github.com/zowe/zowe-cli-db2-plugin) | [zowe-cli-db2-plugin pipeline](https://github.com/zowe/zowe-cli-db2-plugin/blob/master/Jenkinsfile) | 
+| zowe-cics-*.tgz |  IBM® CICS® Plug-in for Zowe CLI | [zowe-cli-cics-plugin](https://github.com/zowe/zowe-cli-cics-plugin) | [zowe-cli-cics-plugin pipeline](https://github.com/zowe/zowe-cli-cics-plugin/blob/master/Jenkinsfile) |
+| zowe-db2-*.tgz | Zowe CLI Plug-in for IBM® Db2® for z/OS® | [zowe-cli-db2-plugin](https://github.com/zowe/zowe-cli-db2-plugin) | [zowe-cli-db2-plugin pipeline](https://github.com/zowe/zowe-cli-db2-plugin/blob/master/Jenkinsfile) |
 | zowe-ims-*.tgz | IBM® IMS™ Plug-in for Zowe CLI | [zowe-cli-mq-plugin](https://github.com/zowe/zowe-cli-mq-plugin) | [zowe-cli-mq-plugin](https://github.com/zowe/zowe-cli-mq-plugin/blob/master/Jenkinsfile) |
```
It looks like some of these rows are misaligned; i.e. zowe-scs shows an ims pipeline text, zowe-ims shows an mq plugin

BrandonJenkins14 replied:
my mistake, will resolve today!

BrandonJenkins14 commented:
This release is waiting for: 
- @latest TPSR scan to be updated and functional
- @latest CLI package added to Zowe.org

BrandonJenkins14 commented:
Closing this PR. We will reintroduce the Active Development doc version at a later time.
